vm/src/stdlib/json.rs

```diff
@@ -42,7 +42,7 @@ pub fn json_loads(obj: PyObjectRef, vm: &VirtualMachine) -> PyResult {
         let module = vm
             .get_attribute(vm.sys_module.clone(), "modules")
             .unwrap()
-            .get_item("json", vm)
+            .get_item("_serde_json", vm)
```

Comment:
do we include this version into binary, but never using?

Reply:
Yeah, I suppose there really isn't a point to keeping it around; most of the functionality is in `py_serde.rs` anyway.

Reply:
Not sure about the future - but If we are going to *incubate* rust native libraries, probably we can grow up the rust version of library from this one. Is this not related to `_json` in CPython?

Reply:
No, `_json` has low-level json operations, e.g. "find where this string ends", "convert this string to JSON", or streaming encoding/lexing classes.

Reply:
We could find another json library, but serde_json doesn't expose any lexing functionality.